manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.yaml

```
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient

```

```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```
You'll also have to update the files to reflect the name change.